securitycenter/assets/add_delete_security_marks.go

```diff
@@ -14,6 +14,7 @@
 
 package assets
 
+// [START securitycenter_add_delete_security_marks]
```

Comment:
Is this the right prefix? The snippet-bot doesn't seem to like it. Should it be `scc`?

Reply:
Yes it's correct. Once all offending snippets are wrapped, I'll be moving the prefix to `securitycenter` (in app), then fix doc includes to reference the new prefix. After all of that is submitted, the old "inside" tags can be removed. Snippet bot will not like any of these for now.